index.html

```diff
@@ -68,54 +68,59 @@ <h2>@ Tvrdjava, Novi Sad, Serbia</h2>
       <div class="containermp">
         <!-- First row -->
         <div class="first-row">
-          <div class="text"><h1">Main Program</h1></div>
+          <div class="text"><h1>Main Program</h1></div>
```

Comment:
- [ **OPTIONAL** ] For best semantics reasons, I think it would be better if you use a `hx` tag different from an `h1` such as `h2, h3,...`. Your project should include only one `h1` which is your hero text, **"EXIT Music Festival Novi Sad 2022"**. I can suggest you apply this approach everywhere you use the `h1` tag i.e. except on the hero text, **"EXIT Music Festival Novi Sad 2022"** for both pages(HOME && ABOUT)